mod/account/set/right/password.rb

```diff
@@ -5,7 +5,8 @@
 PASSWORD_REGEX = {
   lower: /[a-z]/,
   upper: /[A-Z]/,
-  special_char: /[!\"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~]/,
+  # special_char: /[!"#$%&'()*+,\-\.\/:;<=>?@\[\]\\^_`{|}~]/,
+  special_char: /[!"#$%&'()*+]/,
```

Comment:
Actually, my solution wasn't working. I don't have time to look at this right now. @cschweipert: Can you see if you can get a version that (a) works, (b) passes code climate, and (c) passes semaphore tests?

For now I just deleted a bunch of characters so that the main branch won't be broken.